src/xrEngine/device.cpp

```diff
@@ -34,6 +34,7 @@ ENGINE_API BOOL g_bRendering = FALSE;
 
 BOOL g_bLoaded = FALSE;
 ref_light precache_light = 0;
+int g_dwFPSlimit = 120;
```

Comment:
@revolucas 

> High framerates do not damage GPUs unless they are not properly cooled, in which case any kind of heavy load will damage them.

That's just my argument against a global FPS lock. I think we should go with the fix @Xottab-DUTY proposed and force V-Sync in the main menu.